packages/cli-kit/src/private/node/ui/alert.tsx

```diff
@@ -15,6 +16,10 @@ const typeToLogger: {[key in AlertProps['type']]: Logger} = {
   success: consoleLog,
 }
 
+export interface AlertOptions extends AlertProps {
+  renderOptions?: RenderOptions
```

Comment:
Note that this one leaks a type from Ink through the public interface. We can trust Ink in that they won't introduce breaking changes there (which translate to breaking changes in our APIs) or not keep the trust low and export our type instead that maps to theirs. I'm inclined to the cautious because a breaking change might cause the CLI to blow up on the user end.

Reply:
Not sure I understand 100%, but if we introduced our own type and Ink created a breaking change, wouldn't we need to update our type anyway to adapt to the new interface? Unless we can somehow reproduce the previous version's logic I don't think it's something we can avoid. Also this is an optional attribute exposed by a private module so I wouldn't be too worried about that.